parser/hcl2/convert_test.go

```diff
@@ -129,22 +127,13 @@ func TestConversion(t *testing.T) {
 	}
 	for name, tc := range testTable {
 		bytes := []byte(tc.input)
-		conf, diags := hclsyntax.ParseConfig(bytes, "test", hcl.Pos{Byte: 0, Line: 1, Column: 1})
-		if diags.HasErrors() {
-			t.Errorf("Failed to parse config: %v", diags)
-		}
-		converted, err := convertFile(conf)
-
-		if err != nil {
-			t.Errorf("Unable to convert from hcl: %v", err)
-		}
 
-		jb, err := json.MarshalIndent(converted, "", "\t")
+		json, err := convert.Bytes(bytes, "", convert.Options{})
```

Comment:
Why is the test not excercising the interface of the Parser? I guess originally we needed these tests to check the custom HCL logic but right now it would be better to have a simple test checking the Unmarshal method.

Reply:
It probably makes sense to do so. My intent here was to not change the tests to prove the upstream was backwards compatible with what we already had.

I'll be updating this again soon after we figure out https://github.com/tmccombs/hcl2json/pull/20 which will introduce breaking changes anyway. At that point, these tests matter less and we can call the Parser directly.

Thoughts?

Reply:
Ah yeah it makes sense to ensure that it at least works with the previous HCL2 Parser. That makes sense. Let's keep the tests for now then.